templates/endpoints/models/meta-llama--Llama-2-13b-chat-hf.yaml

```diff
@@ -0,0 +1,47 @@
+# This model requires HUGGING_FACE_HUB_TOKEN environment variable to be set to a Hugging Face
+# Access Token for an account with permissions to download the model.
+deployment_config:
+  autoscaling_config:
+    min_replicas: 1
+    initial_replicas: 1
+    max_replicas: 1
+    target_num_ongoing_requests_per_replica: 400.0
+    metrics_interval_s: 10.0
+    look_back_period_s: 30.0
+    smoothing_factor: 1.0
+    downscale_delay_s: 300.0
+    upscale_delay_s: 60.0
+  max_concurrent_queries: 2000
+  ray_actor_options:
+    resources:
+      accelerator_type_a10: 0.01
+engine_config:
+  model_id: meta-llama/Llama-2-13b-chat-hf
+  type: TextGenerationInferenceEngine
+  model_init_kwargs:
+    trust_remote_code: true
+  scheduler:
+    policy:
+      max_batch_prefill_tokens: 9200
+      max_input_length: 4090
+      max_total_tokens: 4096
+  generation:
+    generate_kwargs:
+      do_sample: true
+      temperature: 0.4
+      top_p: 1.0
+      ignore_eos_token: false
+    prompt_format:
+      system: "[INST] <<SYS>>\n{instruction}\n<</SYS>>\n\n"
+      assistant: " {instruction} </s><s> [INST] "
+      trailing_assistant: " "
+      user: "{instruction} [/INST]"
+      default_system_message: "You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe.  Your answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure that your responses are socially unbiased and positive in nature.\n\nIf a question does not make any sense, or is not factually coherent, explain why instead of answering something not correct. If you don't know the answer to a question, please don't share false information."
```

Comment:
lets keep default message consistent for both models.

Reply:
which one is right @gvspraveen ? the 7b one?